assets/themes/waste/waste.json

```diff
@@ -0,0 +1,31 @@
+{
+  "id": "waste",
+  "maintainer": "",
+  "version": "2022-01-19",
+  "language": [
+    "en",
+    "nl"
+  ],
+  "title": {
+    "en": "Waste",
+    "nl": "Afval"
+  },
+  "description": {
+    "en": "Map showing waste baskets and recycling facilities.",
+    "nl": "Kaart met afvalbakken en recyclingfaciliteiten."
+  },
+  "icon": "./assets/layers/recycling/recycling-14.svg",
+  "startZoom": 19,
+  "startLat": 53.24865,
+  "startLon": 6.60075,
+  "layers": [
+    {
+      "builtin": "waste_basket",
+      "override": {
+        "minzoom": 12
```

Comment:
I would hide the waste basket at lower zoom levels, and only opt to show them at >16. Right now, they clutter the map.

As user of the 'waste'-map, I'm interested in where I can drop my glass/metal/green_waste on city scale. If I'm looking for a waste basket, I'd have zoomed in a whole lot more to a waste basket in my vicinity.

Reply:
Good point, minzoom is now set to 18, feels more reasonable for me

Reply:
Yes, way better!